source/core/src/main/com/deco2800/game/ui/terminal/commands/StartTestCutSceneCommand.java

```diff
@@ -0,0 +1,23 @@
+package com.deco2800.game.ui.terminal.commands;
```

Comment:
was this file used for testing purposes? If yes, will it still be required?

Reply:
This was used for testing purposes, and is still required as it provides a convenient trigger for a cutscene at any point in the game